src/main/java/frc/robot/commands/vision/TestVisionDistance.java

```diff
@@ -28,11 +32,13 @@ public class TestVisionDistance extends CommandBase{
   // How many seconds between each data print
   private static final double kPrintDelay = 0.25;
 
-  public TestVisionDistance(double speed, Drivetrain drive, Vision vision){
+  public TestVisionDistance(double speed, Drivetrain drive, Vision vision, ShuffleboardTab visionTab){
     addRequirements(drive);
     m_drive = drive;
     m_speed = speed;
     m_vision = vision;
+    m_outputString = "Run the vision distance test";
+    visionTab.addString("Distance Test Results", ()->getOutputString());
```

Comment:
It would be better to put number on shuffle board rather than a string wich is difficult to read in the shuffl eboar formant and loses features such as graphing spcific to numbers